src/tools/tidy/src/deps.rs

```diff
@@ -208,6 +208,7 @@ const PERMITTED_RUSTC_DEPENDENCIES: &[&str] = &[
     "byteorder", // via ruzstd in object in thorin-dwp
     "cc",
     "cfg-if",
+    "cfg_aliases",
```

Comment:
Judging from the comments above, this will need compiler team approval. Any idea what the process is to get that approval? @oli-obk hoe does this work? :)

Reply:
it's a widely used crate with MIT license, looks all good

Reply:
Ping @oli-obk 

> I meant to ask about all the new dependencies, not just the one. :)

Reply:
Same reasoning for the others